mne/realtime/base_client.py

```diff
@@ -3,13 +3,13 @@
 #
 # License: BSD (3-clause)
 
-import copy
+import threading
```

Comment:
I used thread for similar function and never had any issues. Yes python has problem with the GIL, but that does not mean thread should never be used. What you want to avoid is having a thread that never release the GIL (i.e. continuously run). For example, when using LSL.pull_sample() or a socket.recv, those function will block until you receive data but this is made with a call to a C extension or to a shared library that will effectively release the GIL during the time it wait.